addons/website/data/website_contactus_data.xml

```diff
@@ -1,6 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <template id="website.contactus" name="Contact Us">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <div class="oe_structure">
```

Comment:
In this commit, you move/rename a file (already renamed originally from website_form) and modify it at the same time. So forward-port and history etc might fail (this is the whole point of your first commit) -> rename it directly like in this third commit but in the first commit (so that the third commit is only about moving code inside).

Note: here GitHub understands this is still a move despite the configuration -> that might not be the case for all git configurations

Reply:
If this comment is about `[MOV] website: move website contactus into separate file` commit, there is no change, isn't it?
It is just a move because later commit actually change the code.
I asked for this commit to avoid avoid the move in the next one, see https://github.com/odoo/odoo/pull/69888#pullrequestreview-661084659

Reply:
In that commit: :wink: 

![Screenshot from 2021-06-07 16-29-14](https://user-images.githubusercontent.com/10338094/121034937-b717ec00-c7ad-11eb-8d77-4bdce775077b.png)

Reply:
Ok I see, I guess it is how github show "part move" instead of complete move :shrug: as there is not a single change/diff here

Reply:
When I say "move", I am talking about renaming a file path ;) Moving code from a file to another is a "change".